commands/createfixpullrequests.go

```diff
@@ -199,6 +199,21 @@ func (cfp *CreateFixPullRequestsCmd) fixSinglePackageAndCreatePR(impactedPackage
 		return err
 	}
 
+	if params.WorkingDirectory != "" {
+		// 'CD' into the relevant working directory
+		restoreDir, e := utils.Chdir(params.WorkingDirectory)
+		if e != nil {
+			err = e
+			return
+		}
+		defer func() {
+			e := restoreDir()
+			if err == nil {
+				err = e
+			}
+		}()
+	}
+
```

Comment:
Can we do all Git commands in the same directory? I mean - would it work if we put this section in the beginning of fixSinglePackageAndCreatePR?

Reply:
I believe it won't work because the .git folder is in the root